### PR TITLE
Add licensing info

### PR DESCRIPTION
**Pull request by Steve-Mcl**
## Description

Adds initial licensing info to docs

## Related Issue(s)

#1745 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [x] Concepts
 - [-] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [-] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
@knolleary first stab at adding licensing info.

I am more than certain much of this will need rewording, restructuring, improving - this 1st iteration should be considered more of a placeholder for the final doc improvements.

**Review by ZJvandeWeg — APPROVED** [review]

File: docs/admin/licensing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# FlowForge Licensing
+
+## Open-Source
+
+FlowForge is Open Source and is free to use.
```
Comment (ZJvandeWeg) [inline docs/admin/licensing.md:5]:
```suggestion
FlowForge's open-source edition is free to use.
```

File: docs/admin/licensing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+It is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+## Features
```
[the remark this answers is not in the document]
Comment (Steve-Mcl) [reply docs/admin/licensing.md:8]: @ZJ, I decided to leave the text as was but added the link to pricing (for up-to-date info)

Resolved in [dcded45](https://github.com/flowforge/flowforge/pull/1848/commits/dcded4595e679b96dde8189743a04a92c20c8292) 

I will re-request review shortly.

Thanks.

File: docs/admin/README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ navTitle: Administering FlowForge
  - [FlowForge Concepts](../user/concepts.md)
  - [Usage Telemetry](/docs/admin/telemetry.md)
  - [Single-Sign On](/docs/admin/sso/README.md)
+ - [Licensing](/docs/admin/sso/licensing.md)
```
Comment (knolleary) [inline docs/admin/README.md:16]:
```suggestion
 - [Licensing](/docs/admin/licensing.md)
```

**Review by knolleary — APPROVED** [review]
Approving to get this into 1.7. Made a fix to a broken link, but otherwise, we can iterate if any further changes are needed. Better to have something in than nothing.

**Pull request merged**